stacks-signer/src/signer.rs

```diff
@@ -279,19 +279,13 @@ impl From<SignerConfig> for Signer {
             signer_config.signer_entries.public_keys,
         );
 
-        if let Some(encrypted_state) = stackerdb
-            .get_encrypted_signer_state(signer_config.signer_slot_id.into())
-            .expect("Failed to load encrypted signer state from StackerDB")
-        {
-            let serialized_state = decrypt(&state_machine.network_private_key, &encrypted_state);
-            let state = serde_json::from_slice(&serialized_state)
-                .expect("Failed to deserialize decryoted state");
-            debug!(
-                "Reward cycle #{} Signer #{}: Loading signer",
-                signer_config.reward_cycle, signer_config.signer_id
-            );
-            state_machine.signer = v2::Signer::load(&state);
-        }
+        if let Some(state) = load_encrypted_signer_state(
+            &mut stackerdb,
+            signer_config.signer_slot_id.into(),
+            &state_machine.network_private_key,
+        ).or_else(|err| {warn!("Failed to load encrypted signer state from StackerDB, falling back to SignerDB: {err}"); load_encrypted_signer_state(&signer_db, signer_config.reward_cycle, &state_machine.network_private_key)}).expect("Failed to load encrypted signer state from both StackerDB and SignerDB") {
```

Comment:
nit: formatting

Reply:
Oh, I wonder why this didn't get formatted automatically :thinking: `cargo fmt-stacks` failed to get to this for some reason.

I've manually formatted this now. Thanks for spotting :pray: